Updates for contributor plus (& more) (#6797)
Expensify/App: CONTRIBUTING.md

Welcome! Thanks for checking out the new Expensify app and for taking the time to contribute!

## Getting Started

> If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/App/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, our app philosophy, etc). Please read both documents before asking questions, as it may be covered within the documentation.

Reviewer: `app philosophy` is kinda ambiguous here. I'd just say `best practices` maybe (unless you wanted something more general).

Reviewer: ahhh I like this more! App philosophy seems like it encompasses those other facets.

Author: Nice, agreed!

#### Test Accounts

> You can create as many accounts as needed in order to test your changes directly from [the app](https://new.expensify.com/). An initial account can be created when logging in for the first time, and you can create additional accounts by starting a chat with a valid email or phone in the "New Chat" or "Group Chat" input, then tapping the avatar.

Reviewer: ```suggestion
You can create as many accounts as needed in order to test your changes directly from [the app](https://new.expensify.com/). An initial account can be created when logging in for the first time, and additional accounts can be created by using the "New Chat" or "Group Chat" option in the Global Create Menu, inputting a valid email or phone number , and tapping the user's avatar.
```

Reviewer: idk if `Global Create Menu` is a widely accepted term (referring to the floating green `+` button), but I think this explanation is a bit more succinct imo.

Author: +1 -> I kinda like this phrasing even better:

> ... and additional accounts can be created by opening the "New Chat" or "Group Chat" pages via the Global Create menu, inputting a valid email or phone number, and tapping the user's avatar.

Reviewer: Awesome, I think my explanation was a bit unclear there, but this one makes a ton of sense!

Author: Nice! Team work 👍

At this time, we are not hiring contractors in Crimea, North Korea, Russia, Iran, Cuba, or Syria.

## Asking Questions
If you have any general questions, please ask in the #expensify-open-source Slack channel. To request an invite to the channel, just email contributors@expensify.com with the subject `Slack Channel Invite` and include a link to your Upwork profile. We'll send you an invite! Note: The Expensify team will not be able to respond to direct messages in Slack.

If you are hired for an Upwork job and have any job-specific questions, please ask in the GitHub issue or pull request. This will ensure that the person addressing your question has as much context as possible.

## Reporting Vulnerabilities
If you've found a vulnerability, please email security@expensify.com with the subject `Vulnerability Report` instead of creating an issue.

## Payment for Contributions

> We hire and pay external contributors via Upwork.com. If you'd like to be paid for contributing, please create an Upwork account, apply for an available job in [GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22), and finally apply for the job in Upwork once your proposal gets selected in GitHub. If you think your compensation should be increased for a specific job, you can request a reevaluation by commenting in the Github issue where the Upwork job was posted. 

Author: @mallenexpensify how does this phrasing sound for this section?

Reviewer 2: I like it!

1. Check to ensure an issue does not already exist for this topic in the [New Expensify Issue list](https://github.com/Expensify/App/issues). Please use your best judgement by searching for similar titles and issue descriptions.
2. If your bug or enhancement matches an existing issue, please feel free to comment on that GitHub issue with your findings if you think it will help solve the issue.
3. If there is no existing GitHub issue or Upwork job, report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Include all relevant data, screenshots and examples in the post or thread.

> 4. After review in #expensify-open-source, if you provided a quality proposal we choose to implement it, a GitHub issue will be created and your Slack handle will be included in the original post after `Issue reported by:`

Reviewer: ```suggestion
4. After review in #expensify-open-source, if you've provided a quality proposal that we choose to implement, a GitHub issue will be created and your Slack handle will be included in the original post after `Issue reported by:`
```

Author: Nice 👍
